services/motion/builtin/builtin.go

```diff
@@ -321,21 +321,26 @@ func (ms *builtIn) MoveOnGlobe(
 
 		// once execution responds: return the result to the caller
 		case resp := <-ma.responseChan:
-			ms.logger.Debugf("execution complete: %#v", resp)
+			if failed(resp) {
+				ms.logger.Debugf("execution failed: %s", resp)
+				ma.cancel()
+				return resp.success, resp.err
+			}
+			ms.logger.Debug("execution complete successfully")
 			ma.cancel()
 			return resp.success, resp.err
 
 		// if the position poller hit an error return it, otherwise replan
 		case resp := <-moveRequest.position.responseChan:
-			ms.logger.Debugf("position response: %#v", resp)
+			ms.logger.Debugf("position response: %s", resp)
```

Comment:
now that the resp struct has a String method (that gives better error messages) we can print it with `%s`.

Reply:
done